src/main/docker/artemis/Dockerfile

```diff
@@ -0,0 +1,55 @@
+#-----------------------------------------------------------------------------------------------------------------------
+# build stage
+#-----------------------------------------------------------------------------------------------------------------------
```

Comment:
The command to build this docker file is rather awkward since the build context has to be set. Correct me if I'm wrong. I think a simple `docker build .` will not work

Reply:
`docker build -t artemis-local -f src/main/docker/artemis/Dockerfile .` should be a valid docker build command.

Easiest command currently to reuse it in our docker compose universe would be `docker compose build artemis-app`.

I tried to keep the Dockerfile universal. It should in theory even be possible to inject a `.war` file into the second stage, but I have never tried this.

Reply:
Added steps on how to build the Dockerfile rather in file than in the docs after taking a look in the docs.